lib/linguist/languages.yml

```diff
@@ -354,6 +354,16 @@ Awk:
   - nawk
   ace_mode: text
   language_id: 28
+Ballerina:
+  type: programming
+  aliases:
+  - ballerinalang
```

Comment:
@pchaigno Do we need this? I forget what Linguist uses these aliases for - it's either a way to facilitate advanced searches by allowing users to search for languages using a combination of acceptable names.

My other theory is it has something to do with `linguist-languge=` attributes becoming hairy to work with if the name being targeted contains whitespace characters. I haven't had the most amount of fun with working my head around that, myself. 👎  So a name like "4-word company configuration language" can be cut down with a slug-style alias: `4wordcompanyconflang`.

Reply:
I think it's used for both searches and code snippets:
    ```alias

Reply:
@Alhadis: I'll remove this entry.

Reply:
@Shan1024 Great, thank you!

@pchaigno Ah right, of course, I forgot about tagging fenced code-blocks. 😅  Silly me.